internal/pkg/v2/dto/core/data/event.go

```diff
@@ -0,0 +1,41 @@
+package data
+
+import (
+	dtoBase  "github.com/edgexfoundry/edgex-go/internal/pkg/v2/dto/common/base"
+	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
```

Comment:
create a temporary contract package in edgex-go instead of using go-mod-core-contracts, because there are some change in models between v1 and v2

Reply:
Models added under `internal/pkg/v2/model` directory.